_sass/base.scss

```
@@ -50,6 +50,10 @@ md: 720px,
 lg: 960px
 );
 
+/** Icon sizes **/
+.material-icons.md-16 { font-size: 16px; }
```

try this. note the formatting of the css as well 

```suggestion
.material-icons.md-16 { 
  font-size: 16px;
  padding-right: 8px;
  vertical-align: middle;
}
```